src/core/pointcloud/qgspointcloudlayer.cpp

```diff
@@ -839,11 +858,35 @@ void QgsPointCloudLayer::resetRenderer()
   {
     calculateStatistics();
   }
-  if ( mRenderer->type() == QLatin1String( "extent" ) )
+  if ( !mRenderer || mRenderer->type() == QLatin1String( "extent" ) )
   {
     setRenderer( QgsPointCloudRendererRegistry::defaultRenderer( this ) );
   }
   triggerRepaint();
 
   emit rendererChanged();
 }
+
+void QgsPointCloudLayer::saveCopcStatistics()
+{
+  // If the point cloud doesn't have a valide PDAL data provider and a valid index we don't save any statistics
```

Comment:
```suggestion
  // If the point cloud doesn't have a valid PDAL data provider and a valid index we don't save any statistics
```

Reply:
Now that I added 'e' to valid. I feel french :)